wycheproof/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "w2blb"
+version = "0.1.0"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+description = "Utilities for retrieving Wycheproof test vectors"
+repository = "https://github.com/RustCrypto/utils"
+keywords = ["test", "Wycheproof"]
+categories = ["cryptography", "no-std"]
+edition = "2018"
+
+[dependencies]
+blobby = "*"
+hex = "*"
+maplit = "*"
+serde = { version = "*", features = ["derive"] }
+serde_json = "*"
+structopt = "*"
```

Comment:
`structopt` is a bit heavy dependency. Since this utility is not for public usage, I think something like this should be fine:
```rust
let args: Vec<String> = std::env::args().collect();
let wycheproof_dir = args.get(1).expect("Provide directory with wycheproof vectors");
let algorithm = args.get(2).expect("Provide algorithm family");
let out_path = args.get(3).expect("Provide path for output blobby file");
let descriptions_path = args.get(4).expect("Provide path for descriptions file");
```
Also for consistency with other toml files please keep the final newline character.

Reply:
Done.